src/translations/nl.json

```diff
@@ -1,11 +1,6 @@
 {
-  "boilerplate.components.Footer.author.message": "Met liefde gemaakt door {author}.",
-  "boilerplate.components.Footer.license.message": "Dit project valt onder Mozilla Public License 2.0.",
-  "boilerplate.components.Header.features": "",
-  "boilerplate.components.Header.home": "",
-  "boilerplate.containers.HomePage.start_project.header": "Welkom bij de Signalen Informatievoorziening Amsterdam",
-  "boilerplate.containers.HomePage.start_project.message": "Ontwikkeld door Datapunt Amsterdam",
-  "boilerplate.containers.LocaleToggle.nl": "",
-  "boilerplate.containers.LocaleToggle.en": "",
-  "boilerplate.containers.NotFoundPage.header": "Pagina niet gevonden."
+  "boilerplate.containers.NotFoundPage.header": "Pagina niet gevonden.",
+  "menu.afhandelen": "Afhandelen",
```

Comment:
What if the string 'Afhandelen' is used in a component other than `Menu`? The prefix `menu` doesn't make sense there and, in the future, can lead to duplicate translations because of that. I would choose a context independent prefix, if at all.

Reply:
The idea of these translation is to target a specific use right? So the string "Afhandelen" might occur somewhere else as well. My sense is that we want it to be declared multiple times. Because the contexts are different which means the meaning may be different.

So having a `menu.afhandelen` makes sense. As well as maybe a button somewhere named: `"admin.melding.detail.afhandelen": "Afhandelen"`